Clarifying How to Generate API Key (#470)
magento/merchdocs: src/configuration/services/saas.md

Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Services]({% link configuration/services.md %}) >  Magento Services

Magento allows you to link your Magento store to a SaaS service through the Admin UI and an API key.

## Generate an API key {#apikeys}

> When you create a [Magento account]({% link magento/magento-account.md %}), identified by a MageID, you can generate a Magento API key to connect to the SaaS services within Magento. To utilize Magento Commerce SaaS services, such as Product Recommendations, the API key needs to be generated by the license-holder, or if you are a Magento partner, the contract signer generates the API key. Then, the API key can be passed to the systems integrator or development team, who can manage the SaaS projects and environments on behalf of the license-holder.

Reviewer: ```suggestion
When you create a [Magento account]({% link magento/magento-account.md %}), identified by a MageID, you can generate a Magento API key to connect to the SaaS services within Magento. To use Magento Commerce SaaS services, such as Product Recommendations, the license-holder must generate the API key. However, if you are a Magento partner, the contract signer generates the API key. Then, the API key can be passed to the systems integrator or development team that manages the SaaS projects and environments on behalf of the license-holder.
```

> When you create a [Magento account]({% link magento/magento-account.md %}), identified by a MageID, you can generate a Magento API key to connect to the SaaS services within Magento. The API key needs to be generated by the license-holder. If you are a Magento partner, the contract signer needs to generate the API key. Then, the API key can be passed to the systems integrator or development team, who can manage the SaaS projects and environments on behalf of the license-holder.

Reviewer 2: To utilize Magento Commerce SaaS services, the API key needs to be generated by the license-holder.
